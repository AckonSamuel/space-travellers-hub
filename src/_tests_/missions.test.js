import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../redux/configureStore';
import Mission from '../Components/missions/mission';