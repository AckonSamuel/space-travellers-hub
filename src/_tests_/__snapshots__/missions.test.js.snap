// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A test for missions component Check if it renders correctly 1`] = `
<section
  className="super-mission"
>
  <div
    className="header-container"
  >
    <p>
      Mission
    </p>
    <p
      className="hola"
    >
      Description
    </p>
    <p
      className="status"
    >
      Status
    </p>
    <p
      className="join"
    >
      Join
    </p>
  </div>
  <ul
    className="mission-container>"
  />
</section>
`;

exports[`A test for missions component Check whether mission component renders 1`] = `
<div
  className="mission-table"
  id="heloo"
>
  <h2>
    Thaicom
  </h2>
  <p
    className="mission-description"
  >
    Thaicom is the name of a series of  communications satellites operated from Thailand, and also the name of Thaicom Public Company Limited, which is the company that owns and operates the Thaicom satellite fleet and other telecommunication businesses in Thailand and throughout the Asia-Pacific region. The satellite projects were named Thaicom by the King of Thailand, His Majesty the King Bhumibol Adulyadej, as a symbol of the linkage between Thailand and modern communications technology.
  </p>
  <div
    className="border"
  >
    <p
      className="mission-status"
    >
      Not a member
    </p>
  </div>
  <button
    className="mission-button"
    id="heloo"
    onClick={[Function]}
    type="button"
  >
     Join mission
  </button>
</div>
`;
